javascript/401/README.md

```diff
@@ -0,0 +1,5 @@
+# 401 JavaScript Code Challenges
+
+## Table of Contents
+
+1. [Reverse Array](class-01-reverse-array/README.md)
```

Comment:
Your structure set up looks fine and the first week no actual code is needed to be written!